readme.md

```diff
@@ -228,6 +228,10 @@ Info on a property.
 *   `mustUseProperty` (`boolean`) — If a DOM is used, setting the property
     should be used for the change to take effect (this is true only for
     `'checked'`, `'multiple'`, `'muted'`, and `'selected'`)
+*   `unspecified` (`boolean`) — The property is not in the [list](#support)
```

Comment:
@wooorm I think `unsupported` make more sense?

Reply:
hmm. I kinda feel like we added “support” for these unspecified properties in GH-7.

Reply:
Yes, but with unsupported I mean that we can't provide additional information.
The [GH-7](https://github.com/wooorm/property-information/issues/7) was about to make normalize correctly.